Simplenote/Classes/SPAuthViewController.swift

```diff
@@ -322,14 +320,28 @@ private extension SPAuthViewController {
     }
 
     @objc func passkeyAuthAction() {
-        Task {
-            //TODO: Handle errors
-            //TODO: Handle email not valid
-            passkeyAuthenticator = PasskeyAuthenticator(authenticator: controller.simperiumService)
-            try? await passkeyAuthenticator?.attemptPasskeyAuth(for: email, in: self)
+        guard ensureWarningsAreOnScreenWhenNeeded() else {
+            return
+        }
+
+        Task { @MainActor in
+            lockdownInterface()
+            do {
+                try await attemptPasskeyAuthentication(for: email)
+            } catch {
+                passkeyAuthFailed(error)
+            }
         }
     }
 
+    private func attemptPasskeyAuthentication(for email: String) async throws {
+        let passkeyAuthenticator = PasskeyAuthenticator()
+        let challenge = try await passkeyAuthenticator.fetchAuthChallenge(for: email)
+        let verify = try await passkeyAuthenticator.attemptPasskeyAuth(challenge: challenge, in: self)
+        controller.simperiumService.authenticate(withUsername: verify.username, token: verify.accessToken)
```

Comment:
`fetchAuthChallenge` + `attemptPasskeyAuth` can probably be converged into a single API, making it WAY simpler for the VC